src/components/Avatar.js

```diff
@@ -1,23 +1,45 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './Avatar.less';
 
-const Avatar = ({ username, size }) =>
-  (<img
-    className="Avatar"
-    style={{ minWidth: `${size}px`, width: `${size}px`, height: `${size}px` }}
-    alt={username}
-    src={`${process.env.IMG_HOST}/@${username || 'steemconnect'}?s=${size}`}
-  />);
+class Avatar extends Component {
+  static propTypes = {
+    username: PropTypes.string,
+    size: PropTypes.number,
+  };
 
-Avatar.propTypes = {
-  username: PropTypes.string,
-  size: PropTypes.number,
-};
+  static defaultProps = {
+    username: undefined,
+    size: 34,
+  };
 
-Avatar.defaultProps = {
-  username: undefined,
-  size: 34,
-};
+  constructor(props) {
+    super(props);
+    this.state = {
+      imgUrl: `${process.env.IMG_HOST}/@${props.username || 'steemconnect'}?s=${props.size}`,
+    };
+  }
+
+  onError = () => {
+    this.setState({
+      imgUrl: 'https://res.cloudinary.com/hpiynhbhq/image/fetch/s--hWKFE2sA--/c_fill,h_200,w_200/http://res.cloudinary.com/hpiynhbhq/image/upload/v1501526513/avatar_juzb7o.png',
```

Comment:
Can you use https://res.cloudinary.com/hpiynhbhq/image/upload/v1506948447/p72avlprkfariyti7q2l.png ?

Reply:
yeah just updated @bonustrack